bin/functions

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
+# If a command fails we want to abort right away, and we do not want to use unset variables
+set -euo pipefail
```

Comment:
I appreciate this, but we removed this some time ago in favor of finding unset variables prior to run-time with shellcheck.

As far as having all command failures result in exiting the shell script, I've found this can be useful in quick and dirty scripts, but the larger and more sophisticated the script is, the more likely it is that we are depending on some commands to result in non-zero return codes without the entire script aborting.

Reply:
I added it mainly because the sha256sum failure didn't result in an error. That one was in my opinion quite important. But I'll remove it and add exit codes to all places where I expect exit codes to exist.

Reply:
Thank you!

Reply:
Sometimes you use return 1 and not exit 1. My guess now this was on purpose. What is the difference?